contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3377,3 +3377,30 @@ $$
     SELECT current_setting('application_name');
 $$
 LANGUAGE sql PARALLEL SAFE STABLE;
+
+CREATE OR REPLACE FUNCTION OBJECT_DEFINITION(IN object_id INT)
```

Comment:
Is it supposed to work for cross-db calls also? If yes then using sys views won't be sufficient.

Reply:
No, it should show NULL for cross-db call. It works as expected.